packages/browser/__tests__/Session.spec.ts

```diff
@@ -419,6 +419,29 @@ describe("Session", () => {
         ).toHaveBeenCalledTimes(1);
       });
 
+      it("does not attempt to use the workaround if it is explicitly disabled", async () => {
+        mockLocalStorage({
+          "tmp-resource-server-session-info": JSON.stringify({
+            webId: "https://my.pod/profile#me",
+            sessions: {
+              "https://my.pod/": { expiration: 9000000000000 },
+            },
+          }),
+        });
+        const clientAuthentication = mockClientAuthentication();
+        clientAuthentication.handleIncomingRedirect = jest.fn();
+        const mySession = new Session({ clientAuthentication });
+        await mySession.handleIncomingRedirect({
+          url: "https://some.url",
+          useEssSession: false,
```

Comment:
I suppose another test to verify that it does not attempt to use the workaround if it is enabled but silent authentication is enabled too might be a good addition, or is that very complex due to having to mock silent authentication?

Reply:
Good point, and it's not that difficult to mock: 27cb103ec419270d73d6283dfd389e26b3a70e4a. The assumption is that if the workaround is used, the function returns before calling `ClientAuthentication::handleIncomingRedirect`, the result of which will lead either to being logged in, or to attempt silent authentication. Also, I added tests to verify the value of the variable in local storage.